engine/enginerpc/engine.proto

```diff
@@ -398,6 +398,113 @@ message EngineService {
     UNKNOWN = 3;
   }
   State current_state = 6;
+  string display_name = 7;
+  string check_command = 8;
+  string event_handler = 9;
+  State initial_state = 10;
+  uint32 check_interval = 11;
+  double retry_interval = 12;
+  int32 max_check_attempts = 13;
+  repeated string contactgroups = 14;
+  repeated string contacts = 15;
+  uint32 notification_interval = 16;
+  uint32 first_notification_delay = 17 ;
+  uint32 recovery_notification_delay = 18 ;
+  bool notify_on_unknown = 19 ;
+  bool notify_on_warning = 20 ;
+  bool notify_on_critical = 21 ;
+  bool notify_on_ok = 22 ;
+  bool notify_on_flappingstart = 23 ;
+  bool notify_on_flappingstop = 24 ;
+  bool notify_on_flappingdisabled = 25 ;
+  bool notify_on_downtime = 26 ;
+  bool stalk_on_ok = 27 ;
+  bool stalk_on_unknown = 28 ;
+  bool stalk_on_warning = 29 ;
+  bool stalk_on_critical = 30 ;
+  bool is_volatile = 31 ;
+  string notification_period = 32 ;
+  bool flap_detection_enabled = 33;
+  double low_flap_threshold = 34;
+  double high_flap_threshold = 35;
+  bool flap_detection_on_ok = 36;
+  bool flap_detection_on_warning = 37;
+  bool flap_detection_on_unknown = 38;
+  bool flap_detection_on_critical = 39;
+  int32 process_performance_data = 40;
+  bool check_freshness_enabled = 41;
+  int32 freshness_threshold = 42;
+  bool passive_checks_enabled = 43;
+  bool event_handler_enabled = 44;
+  bool active_checks_enabled = 45;
+  int32 retain_status_information = 46;
+  bool retain_nonstatus_information = 47;
+  bool notifications_enabled = 48;
+  bool obsess_over = 49;
+  string notes = 50;
+  string notes_url = 51;
+  string action_url = 52;
+  string icon_image = 53;
+  string icon_image_alt = 54;
+  bool problem_has_been_acknowledged = 55;
+  enum AckType { 
+    NONE = 0; 
+    NORMAL = 1; 
+    STICKY = 2;
+  }
+  AckType acknowledgement = 56;
```

Comment:
If think there is a redundant information here
problem_has_been_acknowledged is active only if acknowledgement is NORMAL or STICKY.
So I think we can only keep the acknowledgement information.